packages/eas-cli/src/commands/worker/alias.ts

```diff
@@ -0,0 +1,119 @@
+import { Flags } from '@oclif/core';
+import chalk from 'chalk';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import { ExpoGraphqlClient } from '../../commandUtils/context/contextUtils/createGraphqlClient';
+import Log from '../../log';
+import { ora } from '../../ora';
+import { promptAsync } from '../../prompts';
+import {
+  assignWorkerDeploymentAliasAsync,
+  selectWorkerDeploymentOnAppAsync,
+} from '../../worker/deployment';
+
+export default class WorkerAlias extends EasCommand {
```

Comment:
Should we support non-interactive mode in this command as well?

Reply:
Yes, that would be ideal. It's one of the follow-up tasks we have, basically adding `--json` to the `worker:alias` command.

The non-interactive mode is a bit implicit here, if you run with `--id` and `--alias`, it would not ask the user and just set the alias. But, I agree we should add the full `--non-interactive` and `--json` flags here too.